assets/css/admin.css

```diff
@@ -4,6 +4,7 @@
 	gap: 1em;
 	margin-bottom: 1em;
 }
+
```

Comment:
Can we move these CSS changes to a different issue and PR since they're out of scope for this one?

Reply:
Actually it looks like these changes were from another commit already in the `main` branch.